cluster/examples/kubernetes/ceph/csi/template/cephfs/csi-cephfsplugin.yaml

```diff
@@ -22,6 +22,11 @@ spec:
       dnsPolicy: ClusterFirstWithHostNet
       containers:
         - name: driver-registrar
+          # This is necessary only for systems with SELinux, where
+          # non-privileged sidecar containers cannot access unix domain socket
+          # created by privileged CSI driver container.
+          securityContext:
+            privileged: true
```

Comment:
Instead of always running privileged, we could set it to true only when the [ROOK_HOSTPATH_REQUIRES_PRIVILEGED](https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator-openshift.yaml#L290-L291) env var is set to true. We already use this in other places to run mons and other daemons privileged due to selinux/openshift.

Reply:
https://github.com/rook/rook/pull/4639#discussion_r365045870. we cant make this optional I believe as the plugin-container in daemonset  pod will always be privileged due to bidirectional mount